FiskalySDK.xcodeproj/project.pbxproj

```diff
@@ -373,14 +376,10 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
Oh, this might pose an issue for us. We're still actively supporting iOS 10 and iOS 11.

Reply:
This was just one of the 'update to recommended settings' changes from Xcode, so not really necessary for this fix. I'll try changing it back and see if everything still works.

Reply:
The problem with setting the deployment target back to iOS 10.0 is that it would need armv7 support, and I couldn't compile the new version of the client library with armv7 support, since it's written in Go, and Go doesn't support 32-bit since 1.15 as far as I know. Since you are also using Go, have you found a way around that?

Reply:
It is possible to create armv7-compatible libraries running Go 1.13 on an Intel Mac, though I'm not sure about the possibility to use an xcframework (necessary to get the simulator support on M1) when developing for such old platforms. Currently I have a separate [branch](https://github.com/fiskaly/fiskaly-sdk-swift/tree/v2_for_32_bit) (without M1 support) to add features while still supporting 32-bit platforms and old Xcode versions.